test/image/mocks/multicategory-inside-ticks.json

```diff
@@ -0,0 +1,76 @@
+{
+  "data": [
+    {
+      "type": "bar",
+      "x": [
+        ["2017", "2017", "2017", "2017", "2018", "2018", "2018"],
+        ["q1", "q2", "q3", "q4", "q1", "q2", "q3" ]
+      ],
+      "y": [1, 2, 3, 1, 3, 2, 3, 1]
+    },
+    {
+      "type": "bar",
+      "x": [
+        ["2017", "2017", "2017", "2017", "2018", "2018", "2018"],
+        ["q1", "q2", "q3", "q4", "q1", "q2", "q3"]
+      ],
+      "y": [1.12, 2.15, 3.07, 1.48, 2.78, 1.95, 2.54, 0.64]
+    },
+
+    {
+      "type": "bar",
+      "x": [
+        ["2017", "2017", "2017", "2017", "2018", "2018", "2018"],
+        ["q1", "q2", "q3", "q4", "q1", "q2", "q3"]
+      ],
+      "y": [1.12, 2.15, 3.07, 1.48, 2.78, 1.95, 2.54, 0.64],
+      "xaxis": "x2",
+      "yaxis": "y2"
+    },
+    {
+      "type": "bar",
+      "x": [
+        ["2017", "2017", "2017", "2017", "2018", "2018", "2018"],
+        ["q1", "q2", "q3", "q4", "q1", "q2", "q3" ]
+      ],
+      "y": [1, 2, 3, 1, 3, 2, 3, 1],
+      "xaxis": "x2",
+      "yaxis": "y2"
+    }
+  ],
+  "layout": {
```

Comment:
It may be good idea to add `"width": 750` and `"height": 450` layout info.

Reply:
Can I asked why? So that it looks better in the test dashboard?

Reply:
When trying to download the image to show the impact of large ticks on label orientation, I noticed that the result was quite different from what was displayed on the screen.

Reply:
That's expected, label-rotation is computed in pixel-space.